src/packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -81,7 +81,7 @@ export class TSClient implements Generatable {
     const code = `${commonCodeJS({ ...this.options, browser: false })}
 
 const dirnamePolyfill = path.join(process.cwd(), ${JSON.stringify(cwdDirname)})
-const dirname = __dirname.length === 1 ? dirnamePolyfill : __dirname
+const dirname = (__dirname.includes('.next/serverless') || __dirname === '/') ? dirnamePolyfill : __dirname
```

Comment:
```suggestion
const dirname = (__dirname === '/' || __dirname.includes('.next/serverless')) ? dirnamePolyfill : __dirname
```
Maybe more generic case first and special case in second for the condition?

Reply:
also matching `'/'` makes me think that it could be a problem on windows, maybe revert to length === 1?

Reply:
There's also `require('path').sep` I believe.

Reply:
Was reverted to `__dirname.length === 1`